app/src/main/java/com/nextcloud/talk/webrtc/WebSocketInstance.kt

```diff
@@ -138,7 +139,7 @@ class WebSocketInstance internal constructor(
             isConnected = false
             messagesQueue = ArrayList()
         }
-        restartWebSocket()
```

Comment:
I am not sure about this one. When we hit the `onFailure` event we call `closeWebSocket`. Doesn't it make sense to then restart the connection? In my tests `onClosed` was not called when we close the webSocket ourselves?!

Reply:
insterted the restartWebSocket() again and will do the delay reconnection part later :+1: